i18n/en-US.properties

```diff
@@ -1086,6 +1086,8 @@ boxui.presence.timeSinceLastCommented = Commented {timeAgo}
 boxui.presence.timeSinceLastModified = Edited {timeAgo}
 # Description for when someone last previewed a document, {timeAgo} is a relative time like 2 months ago
 boxui.presence.timeSinceLastPreviewed = Previewed {timeAgo}
+# Description of the button to toggle the presence overlay
+boxui.presence.toggleButtonLabel = Presence Activity
```

Comment:
Would this string make sense in context? Something like Recent Activity (the title of the list that opens up) might be more clear.